src/App.vue

```diff
@@ -1,259 +1,269 @@
-        <button @click="addEdge">Add edge</button>
-        <button @click="resetNetwork">Reset Network</button>
-        <button @click="removeNode">Remove Node</button>
-        <button @click="removeEdge">Remove Edge</button>
-        <div class="events"><p>Network events: <br /> {{networkEvents}}</p></div>
-        
-        <hr>
-        
-        <h2>Timeline</h2>
-        <timeline ref="timeline" :items="timeline.items" :groups="timeline.groups" :options="timeline.options"></timeline>
+	<div class="wrapper">
+		<h2>Network</h2>
+		<network 
+			class="network"
+			ref="network" 
+			:nodes="network.nodes" 
+			:edges="network.edges" 
+			:options="network.options"
+			@click="networkEvent('click')"
+			@doubleClick="networkEvent('doubleClick')"
+			@oncontext="networkEvent('oncontext')"
+			@hold="networkEvent('hold')"
+			@release="networkEvent('release')"
+			@select="networkEvent('select')"
+			@selectNode="networkEvent('selectNode')"
+			@selectEdge="networkEvent('selectEdge')"
+			@deselectNode="networkEvent('deselectNode')"
+			@deselectEdge="networkEvent('deselectEdge')"
+			@dragStart="networkEvent('dragStart')"
+			@dragging="networkEvent('dragging')"
+			@dragEnd="networkEvent('dragEnd')"
+			@hoverNode="networkEvent('hoverNode')"
+			@blurNode="networkEvent('blurNode')"
+			@hoverEdge="networkEvent('hoverEdge')"
+			@blurEdge="networkEvent('blurEdge')"
+			@zoom="networkEvent('zoom')"
+			@showPopup="networkEvent('showPopup')"
+			@hidePopup="networkEvent('hidePopup')"
+			@startStabilizing="networkEvent('startStabilizing')"
+			@stabilizationProgress="networkEvent('stabilizationProgress')"
+			@stabilizationIterationsDone="networkEvent('stabilizationIterationsDone')"
+			@stabilized="networkEvent('stabilized')"
+			@resize="networkEvent('resize')"
+			@initRedraw="networkEvent('initRedraw')"
+			@beforeDrawing="networkEvent('beforeDrawing')"
+			@afterDrawing="networkEvent('afterDrawing')"
+			@animationFinished="networkEvent('animationFinished')"
+			@configChange="networkEvent('configChange')"
+		>
+		</network>
+		<button @click="addNode">Add node</button>
+		<button @click="addEdge">Add edge</button>
+		<button @click="resetNetwork">Reset Network</button>
+		<button @click="removeNode">Remove Node</button>
+		<button @click="removeEdge">Remove Edge</button>
+		<div class="events"><p>Network events: <br /> {{networkEvents}}</p></div>
 
-        <hr>
+		<hr>
 
-        <h2>Graph2d</h2>
-        <graph-2d ref="graph2d" :items="graph2d.items" :groups="graph2d.groups" :options="graph2d.options"></graph-2d>
-        
-    </div>
+		<h2>Timeline</h2>
+		<timeline ref="timeline" :items="timeline.items" :groups="timeline.groups" :options="timeline.options"></timeline>
+
+		<hr>
+
+		<h2>Graph2d</h2>
+		<graph-2d ref="graph2d" :items="graph2d.items" :groups="graph2d.groups" :options="graph2d.options"></graph-2d>
+
+	</div>
 </template>
 
 <script>
+import Timeline from "./components/Timeline.vue";
```

Comment:
Throughout the package, JS code is preferably single quoted as per https://github.com/airbnb/javascript#strings--quotes. We shall keep a consistency here as well in order to gain readability.

Reply:
My bad. I used auto formatting in a hurry and didnt notice it changed all single quotes to double. I pushed the fix.